lightwood/encoder/time_series/helpers/common.py

```diff
@@ -92,7 +92,7 @@ def get_group_matches(data, combination):
     if isinstance(data['data'], np.ndarray) and len(data['data'].shape) < 2:
         data['data'] = np.expand_dims(data['data'], axis=1)
 
-    if not combination:
+    if not combination or combination == '__default':
```

Comment:
this one is awkward but I can't come up with a better arg. Is there a reason __default is diff than False?